jwst/jump/tests/test_jump_step.py

```diff
@@ -7,13 +7,6 @@
 
 from jwst.jump import JumpStep
 
-try:
```

Comment:
Were there other modules that also had this conditional import? Or are they over in stcal?

Reply:
yes the conditional import is in `stcal`, so installing it as part of `jwst` should prevent the "no opencv installed" warning from occurring upstream

Reply:
So in other words, even though the conditional import may no longer be necessary over in stcal, it doesn't hurt to leave it in there, because the attempted import will succeed and everything will be good.